### ISSUE TITLE
Kartennummern werden nicht berechnet

### ISSUE THREAD

**Issue by maede97** [cevi/automatic_walk-time_tables#383]
Auszug aus einem Cypress Run:
```
awt-swiss-tml-api-1         | Fatal Python error: Segmentation fault
awt-swiss-tml-api-1         | 
awt-swiss-tml-api-1         | Thread 0x00007fd75d11b700 (most recent call first):
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 81 in _worker
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 975 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 1038 in _bootstrap_inner
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 995 in _bootstrap
awt-swiss-tml-api-1         | 
awt-swiss-tml-api-1         | Current thread 0x00007fd75e120700 (most recent call first):
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/rtree/index.py", line 956 in _nearest_obj
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/rtree/index.py", line 1028 in nearest
awt-swiss-tml-api-1         |   File "/app/swiss_TML_api/map_numbers/map_numbers_fetcher.py", line 72 in fetch_map_numbers
awt-swiss-tml-api-1         |   File "/app/app.py", line 107 in get_map_numbers
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 1469 in dispatch_request
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 1484 in full_dispatch_request
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 2190 in wsgi_app
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/flask/app.py", line 2213 in __call__
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/gthread.py", line 334 in handle_request
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/gthread.py", line 282 in handle
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 58 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/concurrent/futures/thread.py", line 83 in _worker
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 975 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 1038 in _bootstrap_inner
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 995 in _bootstrap
awt-swiss-tml-api-1         | 
awt-swiss-tml-api-1         | Thread 0x00007fd75f12f700 (most recent call first):
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 324 in wait
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 622 in wait
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/tqdm/_monitor.py", line 60 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 1038 in _bootstrap_inner
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/threading.py", line 995 in _bootstrap
awt-swiss-tml-api-1         | 
awt-swiss-tml-api-1         | Thread 0x00007fd7644a5740 (most recent call first):
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/selectors.py", line 468 in select
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/gthread.py", line 211 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/base.py", line 142 in init_process
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/workers/gthread.py", line 95 in init_process
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 609 in spawn_worker
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 642 in spawn_workers
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 571 in manage_workers
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/arbiter.py", line 202 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/app/base.py", line 72 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/app/base.py", line 236 in run
awt-swiss-tml-api-1         |   File "/usr/local/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 67 in run
awt-swiss-tml-api-1         |   File "/usr/local/bin/gunicorn", line 8 in <module>
awt-swiss-tml-api-1         | 
awt-swiss-tml-api-1         | Extension modules: markupsafe._speedups, shapely.speedups._speedups, charset_normalizer.md, requests.packages.charset_normalizer.md, requests.packages.chardet.md, fiona._err, fiona._geometry, fiona._shim, fiona._env, fiona.schema, fiona.ogrext, fiona._crs (total: 12)
Error: kend-1               | [ERROR] Connection error while fetching map numbers from awt-swiss-tml-api
Error: ss-tml-api-1         | [2024-06-02 14:11:55 +0000] [6] [ERROR] Worker (pid:7) was sent code 139!
```

**Comment by maede97** [comment]
This is still an issue. We should add a check to cypress to call the API endpoint directly and verify if it works with a given coordinate.
I guess the index is somehow broken?

**Comment by maede97** [comment]
Also recreating the index on docker boot does not work somehow?

**Comment by maede97** [comment]
See c646e0f8af9dd8a0dd14d72e9fdd21d986d10911

**Comment by maede97** [comment]
Fixed in 6.1.0.
